src/core/Note.cpp

```diff
@@ -254,10 +254,13 @@ std::optional<NoteBounds> boundsForNotes(const NoteVector& notes)
 
 	for (const Note* note: notes)
 	{
+		// Treat beat notes as 1 tick long
```

Comment:
Why 1 tick?

Reply:
Even if they are drawn as 4 or something, they can be shorter than that depending on following notes. But they can never be shorter than 1.

If you ask me note->length should return 1 for beat notes, because logically notes must be >0 but it's out of scope and I know there's the note type PR... So I just fix that problem here meanwhile.

The problem being start must be < end for a non-empty bound.